### PR TITLE
Add error msg for no videos found

### PR DESCRIPTION
**Pull request by dalgr**
Possible fix for #707 

I ran cmd:
```
python svtplay-dl https://www.tv4play.se/program/idol/3876494
```
And it works fine with the newest master. But when i run:
```
python svtplay-dl https://www.tv4play.se/program/idol/3876494 -P dash
```
I get no no error message at all. So i added "log.error("No videos found.")"

### CODE REVIEW

File: lib/svtplay_dl/__init__.py [hunk 1]
```diff
@@ -303,8 +303,8 @@ def options_subs_dl(subfixes):
     if options.merge_subtitle and not options.subtitle:
         options_subs_dl(subfixes)
 
-
-    if len(videos) == 0:
```
Comment (dentarg) [inline lib/svtplay_dl/__init__.py]: why the change in logic?

Comment (dalgr) [reply lib/svtplay_dl/__init__.py]: Its an alternative way of checking that 'len(videos) == 0'(length of array, list, tuple). This solution does the same as 'len(videos) == 0' but also include the case that  'videos is None'.

Comment (dentarg) [reply lib/svtplay_dl/__init__.py]: I see, thanks

Comment (dentarg) [reply lib/svtplay_dl/__init__.py]: Can `videos` ever be None? :)

Comment (dalgr) [reply lib/svtplay_dl/__init__.py]: I guess not :)

**Comment by spaam** [comment]
thanks :)
